### DISCUSSION TITLE
Tasks Plugin - Filter by user - For limited users

### DISCUSSION THREAD

**Discussion by SIXASIS** [kimai/Paid-plugins#196] [Questions & Answers]
Im facing an issue where we have limited permissions for some users and when they try to filter their own tasks, its now allowing them , its showing like all the users tasks. 

The only permission that allows this to work which i found like by  brute force, is the "view_all_data" but this doesnt apply in this case because it gives way more acces to those users than expected. 

Is there an specific set of permissions required to let the task filter by user work? 
![image](https://github.com/user-attachments/assets/322203ec-3665-4fb4-bda4-84086d2d78a1)

**Comment by kevinpapst** [comment]
You mean the user cannot select himself in the dropdown?

Works in the demo installation with `john_user` account. See https://www.kimai.org/en/demo.html for credentials and links.

**Comment by SIXASIS** [comment]
The user is able to select himself in the dropdown but when they press Search its not aplying that to the seach it keeps showing all the users tasksSaludos,Ing. Paramá, Nicolás A.El 27 sept 2024, a la(s) 10:47, Kevin Papst ***@***.***> escribió:﻿
You mean the user cannot select himself in the dropdown?
Works in the demo installation with john_user account. See https://www.kimai.org/en/demo.html for credentials and links.

**Reply by kevinpapst** [reply]
Thanks, understood and confirmed. The query tries to be too clever and shows too many tasks.
I created a bug for that: https://github.com/kimai/Paid-plugins/issues/197
